plugins/accelerated-peft/tests/test_peft_plugins.py

```diff
@@ -86,6 +89,45 @@ def test_configure_gptq_plugin():
 
         e.match(f"AutoGPTQAccelerationPlugin: Value at '{key}'")
 
+def test_autogptq_loading():
+    def autogptq_unavailable(package_name: str):
+        return False
+
+    # - Test that error is thrown when use_external_lib is True but no package found.
```

Comment:
it is best to leave very clear comments, I like this
1. Doing this
2. So that this is done
3. and this is checked..

style of comments https://github.com/foundation-model-stack/fms-hf-tuning/blob/9e4b49f25a7d9a383cd8e5dc1e3175e09197f890/tests/acceleration/test_acceleration_framework.py#L357